### DISCUSSION TITLE
致谢署名位置问题

### DISCUSSION THREAD

**Discussion by jiaqiwang969** [sjtug/SJTUThesis#840] [提问]
您好～
请教一下，在致谢时，需要在右下角署名，请问有什么办法规范？如下案例所示。

                                                                                                                           Jiaqi Wang
                                                                                                                   于上海交通大学闵行校区
                                                                                                                         二零二二年十一月

**Comment by LogCreative** [comment]
```latex
\begin{flushright}
Jiaqi Wang \\
于上海交通大学闵行校区 \\
二〇二二年十一月
\end{flushright}
```

> 应当使用 〇 而不是零
